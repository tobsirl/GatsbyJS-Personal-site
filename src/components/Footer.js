import React from 'react';

class Footer extends React.Component {
  render() {
    return (
      <div id="footer">
        <div className="inner">
          <ul className="icons">
            <li>
              <a href="https://github.com/tobsirl" className="icon fa-github">
                <span className="label">Github</span>
              </a>
            </li>
            <li>
              <a
                href="https://www.linkedin.com/in/paul-tobin-irl/"
                className="icon fa-linkedin"
              >
                <span className="label">LinkedIn</span>
              </a>
            </li>
            <li>
              <a href="/" className="icon fa-twitter">
                <span className="label">Twitter</span>
              </a>
            </li>
            <li>
              <a
                href="mailto:tobster111@gmail.com"
                className="icon fa-envelope-o"
              >
                <span className="label">Email</span>
              </a>
            </li>
          </ul>
          <ul className="copyright">
            <li>Paul Tobin 2021 &copy;</li>
            {/* <li>
              Design:
              {' '}
              <a href="http://html5up.net">HTML5 UP</a>
            </li> */}
          </ul>
        </div>
      </div>
    );
  }
}

export default Footer;
