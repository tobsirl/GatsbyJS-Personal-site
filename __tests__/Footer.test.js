import React from 'react';
import renderer from 'react-test-renderer';

import Footer from '../src/components/Footer';

describe('Footer', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Footer siteTitle="Default Starter" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
