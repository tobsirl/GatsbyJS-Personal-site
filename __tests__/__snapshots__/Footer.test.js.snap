// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div
  id="footer"
>
  <div
    className="inner"
  >
    <ul
      className="icons"
    >
      <li>
        <a
          className="icon fa-github"
          href="https://github.com/tobsirl"
        >
          <span
            className="label"
          >
            Github
          </span>
        </a>
      </li>
      <li>
        <a
          className="icon fa-linkedin"
          href="https://www.linkedin.com/in/paul-tobin-irl/"
        >
          <span
            className="label"
          >
            LinkedIn
          </span>
        </a>
      </li>
      <li>
        <a
          className="icon fa-twitter"
          href="/"
        >
          <span
            className="label"
          >
            Twitter
          </span>
        </a>
      </li>
      <li>
        <a
          className="icon fa-envelope-o"
          href="mailto:tobster111@gmail.com"
        >
          <span
            className="label"
          >
            Email
          </span>
        </a>
      </li>
    </ul>
    <ul
      className="copyright"
    >
      <li>
        Paul Tobin 2021 Â©
      </li>
    </ul>
  </div>
</div>
`;
